packages/app/src/admin/InvitePage.tsx

```diff
@@ -42,7 +44,7 @@ export function InvitePage(): JSX.Element {
             } else {
               setResult(response);
             }
-            setEmailSent(body.sendEmail);
+            setEmailSent(body.sendEmail ?? false);
```

Comment:
Nit: You can use `!!` to convert `boolean | undefined` to `boolean`.  We use this pattern throughout the codebase.

`setEmailSent(!!body.sendEmail);`